Makefile

```diff
@@ -57,6 +61,21 @@ lint: fmt vet generate ## All-in-one linting
 generate: ## Generate additional code and artifacts
 	@go generate ./...
 
+.PHONY: install-crd
+install-crd: export KUBECONFIG = $(KIND_KUBECONFIG)
+install-crd: generate ## Install CRDs into cluster
+	kubectl apply -f package/crds
+
+.PHONY: install-samples
+install-samples: export KUBECONFIG = $(KIND_KUBECONFIG)
+install-samples: generate install-crd ## Install samples into cluster
+	kubectl apply -f package/samples
+
+.PHONY: run-operator
+run-operator: export KUBECONFIG = $(KIND_KUBECONFIG)
+run-operator:
```

Comment:
Also missing `kind-setup`

Reply:
Not necessarily in this case. the idea is hat one could run the operator against a live cluster with an exported KUBECONFIG as a form of debugging.

Reply:
Then I'd remove the export, as this indicates it should run against the provided kind. Or maybe do separate targets? One to run it in kind and one to run it in whatever cluster that is configured.

Reply:
added `kind-run-operator`